test/util/scheduler.go

```diff
@@ -0,0 +1,57 @@
+package util
+
+import (
+	"testing"
+
+	"k8s.io/client-go/informers"
+	coreinformers "k8s.io/client-go/informers/core/v1"
+	"k8s.io/client-go/tools/events"
+	"k8s.io/kubernetes/pkg/scheduler"
+	"k8s.io/kubernetes/pkg/scheduler/profile"
+	testutils "k8s.io/kubernetes/test/integration/util"
+)
+
+// InitTestSchedulerWithOptions initializes a test environment and creates a scheduler with default
+// configuration and other options.
+// TODO(Huang-Wei): refactor the same function in the upstream, and remove here.
+func InitTestSchedulerWithOptions(
+	t *testing.T,
+	testCtx *testutils.TestContext,
+	startScheduler bool,
+	opts ...scheduler.Option,
+) *testutils.TestContext {
+	testCtx.InformerFactory = informers.NewSharedInformerFactory(testCtx.ClientSet, 0)
+
+	var podInformer coreinformers.PodInformer
+
+	podInformer = testCtx.InformerFactory.Core().V1().Pods()
+	var err error
+	eventBroadcaster := events.NewBroadcaster(&events.EventSinkImpl{
+		Interface: testCtx.ClientSet.EventsV1beta1().Events(""),
+	})
+
+	opts = append([]scheduler.Option{scheduler.WithBindTimeoutSeconds(600)}, opts...)
```

Comment:
why do we need to explicitly set the bind timeout?

Reply:
copy/paste mistake...